src/features/signup/components/faqList.tsx

```diff
@@ -0,0 +1,38 @@
+import { FAQItem } from "./faqItem";
+
+export const FAQList = () => {
+  return (
+    <div className="mt-28 flex flex-col items-center justify-center">
+      <h1 className="mb-8">FAQ</h1>
+      <FAQItem
+        title="what is the focus of this round?"
+        description="This found is focused on supporting Mexican projects working on reducing the gender gap, collaborating with local communities, and ReFi."
+      />
+      <FAQItem
+        title="Who are the requirements for participation?"
+        description="To participate in this round as a project or as a voter, you must be a Mexican community member and have on-chain proofs, such as: POAPs, Unlock Protocol, NFTs or EAS."
```

Comment:
not sure it should say Mexican community memeber? maybe we can keep it more general "To participate in this round as a project or as a voter, you will need to be approved by the round operators."

Reply:
I actually have a question: what kind of FAQ are we going to provide? since these contents are generated by the design team, and they have this randomly.

Reply:
We should probably provide some general q/a around what rpgf is, what maci is + we can leave the ones you added but with a general answer so whoever forks just changes to what makes sense for their case